src/components/DesktopPatientTable/index.js

```diff
@@ -116,41 +117,32 @@ const desktopViewLabelsForPatientsWithCurrentStats = css`
 `;
 
 const DesktopPatientTable = (props) => {
-  const {
-    selectedCaseData,
-    isShowSpinner,
-    incrementPage,
-    hasNext,
-    selectedCases,
-    page,
-    searchText,
-  } = props;
+  const { selectedCaseData, incrementPage, hasNext } = props;
 
   const dispatch = useDispatch();
   const user = useSelector(getUser);
+  const showSpinner = useSelector(getSpinnerType);
   const wrapperRef = useRef(null);
   const [downloadingPatientId, setDownloadingPatientId] = useState(null);
-  const [isShowLoadMoreSpinner, setIsShowLoadMoreSpinner] = useState(true);
 
   const onCall = useCallback(
     (patientId) => () => handleCallAppointment(dispatch, patientId),
     [dispatch],
   );
 
-  const loadMore = useCallback(() => {
-    if (hasNext && isScrolledToEnd()) {
-      setIsShowLoadMoreSpinner(true);
-      incrementPage();
-    }
-  }, [hasNext, selectedCases, page, searchText]);
-
   const isScrolledToEnd = () => {
     return (
       wrapperRef?.current?.scrollTop >
       wrapperRef?.current?.scrollHeight - wrapperRef?.current?.offsetHeight - 20
     );
   };
 
+  const loadMore = useCallback(() => {
+    if (hasNext && isScrolledToEnd()) {
+      incrementPage();
+    }
+  }, [hasNext, selectedCaseData]);
```

Comment:
if we remove `page` and `searchText` won't this use the old value when incrementing page? Because of callbacks in hooks

Reply:
@aviskarkc10 I have checked behaviour with page and other stuff and without with only selectedCaseData: second approach works correctly unlike the previous. Please test this case to make sure it works correctly